generate.js

```diff
@@ -0,0 +1,42 @@
+const fs = require("fs");
+const path = require("path");
+
+fs.readdir(
+  path.join(__dirname, "profiles"),
+  {
+    withFileTypes: true,
+  },
+  (err, files) => {
+    if (err) {
+      throw err;
+    }
+
+    let profiles = [];
+
+    fs.mkdir(path.join(__dirname, ".cache"), { recursive: true }, () => {});
+
+    files.forEach((file) => {
+      const profile = require(path.join(__dirname, "profiles", file.name));
+
+      profiles = [...profiles, profile];
```

Comment:
```suggestion
    const profiles = [];

    fs.mkdir(path.join(__dirname, ".cache"), { recursive: true }, () => {});

    files.forEach((file) => {
      const profile = require(path.join(__dirname, "profiles", file.name));

      profiles.push(profile);
```

Why not push rather than spread?